src/sql/relation.rs

```diff
@@ -284,19 +285,22 @@ impl<'a, T: QueryToRelationTranslator + Copy + Clone> VisitedQueryRelations<'a,
                     .build();
 
                 // We collect column mapping inputs should map to new names (hence the inversion)
-                let new_columns: Hierarchy<Identifier> =
-                    join.field_inputs().map(|(f, i)| (i, f.into())).collect();
-                let composed_columns = all_columns.and_then(new_columns.clone());
+                let join_columns: Hierarchy<Identifier> =join
+                    .field_inputs()
+                    .map(|(f, i)| (i, f.into()))
+                    .collect();
+                let composed_columns = all_columns.and_then(join_columns.clone());
```

Comment:
Where are you using this ?

Reply:
Not sure to understand, here the only difference from before is that the name of join columns  which I change it from `new_columns` to `join_columns`